benches/benches/bevy_ecs/index/index_iter_indexed.rs

```diff
@@ -0,0 +1,38 @@
+use bevy_ecs::{prelude::*, system::SystemId};
+use core::hint::black_box;
+use glam::*;
+
+const PLANETS: u16 = 1_000;
+const SPAWNS: usize = 1_000_000;
+
+#[derive(Component, Copy, Clone, PartialEq, Eq, Hash)]
+#[component(immutable)]
+struct Planet(u16);
+
+fn find_planet_zeroes_indexed(query: QueryByIndex<Planet, &Planet>) {
+    let mut query = query.at(&Planet(0));
+    for planet in query.query().iter() {
+        let _ = black_box(planet);
```

Comment:
```suggestion
        black_box(planet);
```

(Nit) The `let _ =` isn't necessary, since `black_box()` forces the expression it is passed to be evaluated.

> In practice, black_box serves two purposes:
>
> 1. It prevents the compiler from making optimizations related to the value returned by black_box
> 2. It forces the value passed to black_box to be calculated, even if the return value of black_box is unused

[`black_box()` - How to use this](https://doc.rust-lang.org/beta/std/hint/fn.black_box.html#how-to-use-this)